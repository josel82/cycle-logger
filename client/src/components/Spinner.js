import React from 'react';

const Spinner = () => (
    <div className="spinner-box">
        <div className="spinner"></div>
    </div>
);

export default Spinner;