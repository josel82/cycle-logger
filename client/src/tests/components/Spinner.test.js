import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../components/Spinner';

test('should render spinner correctly', ()=> {
    const wrapper = shallow(<Spinner />);

    expect(wrapper).toMatchSnapshot();
});