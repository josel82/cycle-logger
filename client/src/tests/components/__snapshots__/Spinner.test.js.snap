// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render spinner correctly 1`] = `
<div
  className="spinner-box"
>
  <div
    className="spinner"
  />
</div>
`;
